packages/manager/modules/web-paas/src/components/offer-details/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
+  "web_paas_offer_num_prod_env": "{{ numProdEnvironment }} environnement de production",
+  "web_paas_offer_num_test_env": "{{ numTestEnvironment }} environnement de test",
+  "web_paas_offer_storage_size": "{{ storageSize }} Go de stockage",
+  "web_paas_offer_num_licence": "{{ numLicence }} licence utilisateur"
```

Comment:
i think we should have a translation for `[0;1]` and another one for `[2;∞[`

Examples for French: 
1 environnement de production / 2 environnement`s` de production
1 licence utilisateur / 2 licence`s` utilisateur